src/dev/build/tasks/copy_source_task.ts

```diff
@@ -34,6 +34,7 @@ export const CopySource: Task = {
       '!src/cli*/dev.js',
       '!src/plugins/telemetry/schema/**',
       '!src/core/server/core_app/assets/favicons/favicon.distribution.{ico,png,svg}',
+      '!packages/core/apps/core-apps-server-internal/src/assets/favicons/favicon.distribution.{ico,png,svg}',
```

Comment:
I tried removing the assets from `src/core/server/core_app` but, well, that didn't work.

Reply:
We don't copy packages, so excluding them shouldn't have any effect, can you explain what this is supposed to do?

Reply:
I was purely mimicking whatever had been done in the tasks for where the code was originally.

Reply:
```suggestion
```